pkg/docker/config/config.go

```diff
@@ -719,3 +747,14 @@ func normalizeRegistry(registry string) string {
 	}
 	return normalized
 }
+
+// validateKey verifies that the input key does not have any not allowed prefix
```

Comment:
a nit since you've another
```suggestion
// validateKey verifies that the input key does not have a prefix that is not allowed 
```

Reply:
Fixed :+1: